python/hail/methods/__init__.py

```diff
@@ -3,7 +3,7 @@
     import_interval_list, import_bed, import_fam, grep, import_bgen, import_gen, import_table, \
     import_plink, read_matrix, read_table, get_vcf_metadata, import_vcf, index_bgen
 from .statgen import linreg, sample_rows, ibd, ld_matrix, grm, rrm, pca, hwe_normalized_pca, \
-    pc_relate, split_multi_hts, balding_nichols_model, FilterAlleles
+    pc_relate, split_multi_hts, balding_nichols_model, FilterAlleles, min_rep
```

Comment:
This needs to get added to `index.rst` for methods.

Reply:
Done.